### ISSUE TITLE
Emit a warning when something which is not an exception is caught in an exception handler

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#303]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

```
#!python

try:
    import maybe_missing_2
except 42:
    maybe_missing2 = None
```

The ``except 42` is just silly and Pylint should warn that anything other than an exception was used as an exception in an exception handler.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/303
